content/contributors.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "name": "Jorge Marrero",
+    "position": "React Developer",
+    "company": "Woonivers",
+    "github": "jorgemarrero"
+  }
```

Comment:
@reactcanarias/reacters here we need to add ourselves to appear in Contributors Section